gnucash/gnome-utils/gnc-tree-model-account.c

```diff
@@ -583,7 +584,12 @@ gnc_tree_model_account_compute_period_balance (GncTreeModelAccount *model,
     if (negative)
         *negative = gnc_numeric_negative_p (b3);
 
-    return g_strdup(xaccPrintAmount (b3, gnc_account_print_info (acct, TRUE)));
+    print_info = gnc_account_print_info (acct, TRUE);
+
+    if (print_info.use_symbol)
+        return gnc_wrap_text_with_bidi_ltr_isolate (xaccPrintAmount (b3, print_info));
+    else
+        return g_strdup (xaccPrintAmount (b3, print_info));
```

Comment:
Rather than repeating this test a gazillion times, move it into `XaccPrintAmount`.

Reply:
Can not do that, I did something similar for my first commit but that would affect all the report tests. What I have done is create another function similar to xaccAccountPrint that does do that and used that function where needed. Thinking about it I probably do not need to test for 'use_symbol' as all the places used do display the symbol but no harm in leaving it.

Reply:
Much better, thanks.